.gitignore

```diff
@@ -14,7 +14,7 @@ bootrun.log
 .settings
 .springBeans
 .sts4-cache
-bin/
+#bin/
```

Comment:
before I delete this ... 
I am curious to understand why `bin/` is currently bing ignored?

... and happy to move the `bin/` directory to `scripts/` (or something else).
thank you in advance! 🤓

Reply:
No idea about `bin/`, maybe some tools use it as an output directory?

Regarding the setup of a `scripts` directory, I think that makes sense, and I'd advocate to set it up as a gradle subproject so you can run `shellcheck` and `black` on it on PRs.
https://github.com/DataBiosphere/terra-data-catalog/blob/main/scripts/build.gradle
https://github.com/DataBiosphere/terra-data-catalog/blob/main/.github/workflows/build-and-test.yml#L20

Reply:
we can talk more about this tomorrow at our meeting. 
i'd like to find/create a standard place where people can find these scripts without having to look in different places.